x-pack/plugins/lists/server/services/exception_lists/utils.ts

```diff
@@ -316,61 +304,74 @@ export const transformUpdateCommentsToComments = ({
   existingComments: CommentsArray;
   user: string;
 }): CommentsArray => {
-  const newComments = comments ?? [];
-
-  if (newComments.length < existingComments.length) {
+  if (comments == null && existingComments.length > 0) {
```

Comment:
In the case where two users are editing the same exception item at the same time and they're both adding new comments, we run into trouble here because I think we're relying on the number of `existingComments` and `incomingComments` to be the same minus one? I think we should use the `_version` to detect conflicts before attempting to process the comments.